broker/unified_sql/src/stream_storage.cc

```diff
@@ -228,10 +228,6 @@ void stream::_unified_sql_process_pb_service_status(
                 metric_id, type, pd.value(), pd.unit(), pd.warning(),
                 pd.warning_low(), pd.warning_mode(), pd.critical(),
                 pd.critical_low(), pd.critical_mode(), pd.min(), pd.max());
```

Comment:
case of failure (metric_id undefined) is not managed at the end of if else at line 290